### ISSUE TITLE
[Suggestion] Less obstructive Update notification 

### ISSUE THREAD

**Issue by nofabe** [TeamNewPipe/NewPipe#6413]
Would it be possible to make the notification that an update is available less obstructive? When I open NewPipe, it pops up in the notification bar along with NewPipe's audio controls, which get collapsed and I can't access them without swiping the notification away or swiping into the submenu every time, and as when I click the notification it attempts to open a browser window but it fails anyway, I have to manually go to the github and download the new version, so it'd be nice if it was just like a pop-up in the app or sth

**Comment by skyGtm** [comment]
Looks similar to me #6113

**Comment by nofabe** [comment]
Yes, it's similar, but while he's asking for more notification interaction, I'm asking for it to be made more discreet or given an option to disable it (the latter wouldn't be ideal as I'd still like to know about updates but they're just too much of an annoyance if I don't intend to download it right away, and also since clicking the notification does nothing)... Simply making an in-app pop-up instead of a notification pop-up would be a lot more convenient, if that's not too difficult to implement

**Comment by skyGtm** [comment]
> or given an option to disable it 

There is option to disable it in Settings > Updates


>   and also since clicking the notification does nothing

for me it opens browser and directly attempts to download the NewPipe.apk file. Have you tried using a different browser?

Hope it helps

**Comment by nofabe** [comment]
Ah, my bad, I wasn't aware of that option - as for the pop-up, it does attempt to open a browser window but fails and instantly closes the window... My default browser is Chrome, idk if I have to change that to get it to open in a different browser, it doesn't prompt a choice when clicking the notif

**Comment by skyGtm** [comment]
> it doesn't prompt a choice when clicking the notif

It shows a prompt if you have different browser installed.

If you have other browser installed but it is not showing the prompt to select different browser then:
In your phone System Settings> Apps > find and Select Chrome browser > 'Clear Defaults' [WARNING: NOT Clear Data].

Now click in the Notification again, it should show the prompt
